docs/PUBLISH.md

```diff
@@ -30,9 +30,8 @@ To perform a snapshot release, simply comment `!snapshot` in your PR. Once the p
 Notes:
 
 - Snapshot versions use the following format: `@clerk/package@x.y.z-snapshot.commit`, where `package` is the package name, `x`,`y`,`z` are the major, minor and patch versions respectively, `snapshot` is a stable prerelease mame and `commit` is the id of the last commit in the branch.
-- If you want to name your snapshot release, you can pass an argument to the snapshot comment, eg `!snapshot myname` will use myname instead of `snapshot`, eg: `@clerk/clerk-js@4.1.1-myname.90012`.
```

Comment:
I think this does work 🤔 It just doesn't change the name of the npm tag we publish with.